ios/Classes/Notification.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+class Notification: NSObject {
+    @objc
+    let title: String?;
+    @objc
+    let body: String?;
+    
+    init(title: String?, body: String?) {
+      self.title = title
+      self.body = body
+    }
+    
+    convenience init?(from userInfo: [AnyHashable : Any]) {
+        if let aps = userInfo["aps"] as? [AnyHashable: Any] {
+            if let alert = aps["alert"] as? [AnyHashable: Any] {
+                let title = alert["title"] as? String
+                let body = alert["body"] as? String
+                self.init(title: title, body: body)
+            } else if let title = aps["alert"] as? String {
+                self.init(title: title, body: nil)
+            } else {
+                return nil;
+            }
+        } else {
+            return nil;
+        }
+    }
```

Comment:
it's just a cosmetic change, IMO it looks better than another if. You can use it or discard it if you want - it's not a blocker 
```suggestion
    convenience init?(from userInfo: [AnyHashable : Any]) {
        guard let aps = userInfo["aps"] as? [AnyHashable: Any] else {
            return nil
        }
        
        if let alert = aps["alert"] as? [AnyHashable: Any] {
            let title = alert["title"] as? String
            let body = alert["body"] as? String
            self.init(title: title, body: body)
        } else if let title = aps["alert"] as? String {
            self.init(title: title, body: nil)
        } else {
            return nil
        }
    }
```